### PR TITLE
Adds the tazer drone to the game.

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

SUMMARY: Content "Adds the tazer drone to the game"
#### Purpose of change

I don't think we have any flying non-explosive hacks yet. This change introduces a hack that can zap you to death, is fast, but very fragile.

#### Describe the solution

While this hack is fast, with a good shot you can down this. Even a .22 is enough for the job.

#### Describe alternatives you've considered

I was wondering if we have any ranged drones. Pew pew! 5 bullets only, pistol calibre would be a good start.

#### Testing

Loaded it up in game, checked if it worked.

#### Additional context

https://i.imgur.com/UE6dBtM.gifv

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
ZAPBACK isn't a special attack, it's a monster defence function that only triggers when you hit the monster in melee. This is not a ranged electrical attack. Is this intended?

**Comment by Xenomorph-III** [comment]
The tazer drone only zaps you on contact, and not as a ranged attack.

**Comment by Night-Pryanik** [comment]
What's the purpose of this bot then? It won't zap you if you don't attack it first. It simply flies and do... nothing if you don't disturb it?

**Comment by Xenomorph-III** [comment]
I intended it to chase down a survivor/enemies as soon as something gets into its line-of-sight.

Any way to make it do that?

**Comment by Night-Pryanik** [comment]
> deliver an electric shock

It won't deliver an electric shock until player hits it first. That's how ZAPBACK _defence_ function is working. You need to implement some new _attack_ function for this.

**Comment by Xenomorph-III** [comment]
I added the "TAZER" special attack to it. Now it'll chase down whatever it sees and zap it.

**Comment by Xenomorph-III** [comment]
Now, if I were to make it zap every zero turns...

File: data/json/monsters/drones.json [hunk 1]
```diff
@@ -185,5 +185,20 @@
...
+    "name": { "str": "tazer hack" },
+    "description": "An automated drone, this small quadcopter robot appears to have a tazer strapped to the front. While extrememly fast, it is very fragile.",
+    "hp": 5,
+    "diff": 10,
+    "speed": 300,
+    "color": "cyan",
+    "armor_cut": 2,
+    "armor_bullet": 1,
+    "revert_to_itype": "bot_tazer_hack",
+    "special_attacks": [ [ "ZAPBACK" ], [ "TAZER", 5 ] ]
```
Comment (Night-Pryanik) [inline data/json/monsters/drones.json:202]: Replace `ZAPBACK` special attack with `"special_when_hit": [ "ZAPBACK", 75 ]` line. Enter any value you find appropriate.

**Comment by kevingranade** [comment]
Needs linting.

**Comment by Night-Pryanik** [comment]
Please resolve conflicts and apply linting.

**Comment by Night-Pryanik** [comment]
@Xenomorph-III could you please print and resolve conflicts?

File: data/json/items/generic.json [hunk 2]
```diff
@@ -3284,5 +3284,24 @@
...
+    "color": "green",
+    "name": { "str": "broken tazer hack" },
+    "category": "other",
+    "description": "A broken tazer hack, with its propellers still and tazer inert.  Could be gutted for parts.",
+    "price": 1000,
+    "price_postapoc": 10,
+    "material": [ "steel", "plastic" ],
+    "weight": "10 kg",
+    "volume": "5 L",
+    "length": "20cm",
```
Comment (LaVeyanFiend) [inline data/json/items/generic.json:3301]:
```suggestion
    "longest_side": "20 cm",
```

**Pull request closed without merging**